packages/saltcorn-data/models/config.ts

```diff
@@ -593,6 +593,28 @@ const configTypes: ConfigTypes = {
       "Issued by Apple for enrolled members of the 'Apple Developer Program'." +
       "The team id must be set to build mobile iOS apps that can run on a device.",
   },
+  /** @type {object} */
+  default_files_accept_filter: {
+    type: "String",
+    label: "Default Files accept filter",
+    default: null,
+    blurb: "Specifies a default filter for what file types the user can pick from the file input dialog box. Example is `.doc, text/csv,audio/*,video/*,image/*`",
+  },
+  /** @type {object} */
+  csv_lines_limit: {
```

Comment:
remove this, see comment on table

Reply:
@glutamate, Please see my comments in discussion about table